OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -14,26 +14,26 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>.</OutputPath>
```

Comment:
My bad, that must've happened when I was squashing commits. Monodevelop has a tendency to make stupid changes like this for no reason. I'll get it fixed as soon as I can.